python/nav/web/sortedstats/views.py

```diff
@@ -106,7 +125,11 @@ def collect_result(view, timeframe, rows):
     cache_key = get_cache_key(view, timeframe, rows)
     result = get_result(view, start, end, rows)
     result.collect()
-    cache.set(cache_key, result, timeout=timeout)
+    try:
+        cache = get_cache()
+        cache.set(cache_key, result, timeout=timeout)
+    except InvalidCacheBackendError as e:
+        _logger.error("Error accessing cache for ranked statistics: %s".format(e))
```

Comment:
This log statement appears to use some inconsistent string formatting techniques. As always, when using loggers, I suggest using the built-in string formatting of the logger object, which ensures that time is spent to format the arguments *only if the current log level dictates that this log record should be emitted*:

```suggestion
        _logger.error("Error accessing cache for ranked statistics: %s", e)
```

Reply:
fixed